sway-lib-core/src/ops.sw

```diff
@@ -1019,14 +1018,12 @@ trait OrdEq: Ord + Eq {
     ///     let struct1 = MyStruct { val: 10 };
     ///     let struct2 = MyStruct { val: 10 };
     ///     let result = struct1 >= struct2;
-    ///     assert(result); 
+    ///     assert(result);
     /// }
     /// ```
     fn ge(self, other: Self) -> bool {
         self.gt(other) || self.eq(other)
-    }
-
-    /// Evaluates if one value of the same type is less or equal to than another.
+    }    /// Evaluates if one value of the same type is less or equal to than another.
```

Comment:
There's a missing new line here after the closing brace that the formatter removed. Would you mind making an issue to track it?

Reply:
Please report similar issues. I will fix it now (fix will come in the swayfmt crate)

Reply:
This may have something to do with the newline handler, which is its own traversal of the formatted string in `formatter/mod.rs`